docs/wiki/[Examples]-Create-Custom-Policy-and-Assignments.md

```diff
@@ -0,0 +1,507 @@
+        "scope": "${current_scope_resource_id}",
+        "enforcementMode": null,
+        "nonComplianceMessages": [
+        {
+            "message": "Mandatory tags must be provided."
+        }
+        ]
+    },
+    "location": "${default_location}",
+    "identity": {
+        "type": "SystemAssigned"
+    }
+}
+```
+
+Next create a file named `policy_assignment_es_deny_nic_nsg.json` within the `policy_assignments` directory. Copy the below code in to the file and save it.
+
+### `lib/policy_assignments/policy_assignment_es_deny_nic_nsg.json`
+
+```json
+{
+    "name": "Deny-NIC-NSG",
+    "type": "Microsoft.Authorization/policyAssignments",
+    "apiVersion": "2019-09-01",
+    "properties": {
+      "description": "This policy will prevent NSGs from being applied to network interface cards.",
+      "displayName": "Prevent Network Security Groups from being applied to Network Interface Cards",
+      "notScopes": [],
+      "parameters": {} ,
+      "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policyDefinitions/Deny-NIC-NSG",
+      "scope": "${current_scope_resource_id}",
+      "enforcementMode": null,
+      "nonComplianceMessages": [
+        {
+            "message": "NSGs must not be applied to Network Interface cards."
+        }
+    ]
+
+    },
+    "location": "${default_location}",
+    "identity": {
+      "type": "SystemAssigned"
+    }
+  }
+```
+
+Finally, create a file named `policy_assignment_es_enforce_mandatory_tagging.json`. Copy the below code in to the file and save it.
+
+### `lib/policy_assignments/policy_assignment_es_enforce_mandatory_tagging.json`
+
+```json
+{
+  "name": "Enforce-Mandatory-Tags",
+  "type": "Microsoft.Authorization/policyAssignments",
+  "apiVersion": "2019-09-01",
+  "properties": {
+    "description": "Contains the core policies applicable to the org",
+    "displayName": "Ensure mandatory tagging is applied to both Resources and Resource Groups",
+    "notScopes": [],
+    "parameters": {
+      "EnforceRGTags-Owner": {
+        "Value": "Jane Doe"
+      },
+      "EnforceRGTags-Department": {
+        "Value": "IT"
+      },
+      "EnforceResourceTags-Owner": {
+        "Value": "Jane Doe"
+      },
+      "EnforceResourceTags-Department": {
+        "Value": "IT"
+      }
+    },
+    "policyDefinitionId": "${root_scope_resource_id}/providers/Microsoft.Authorization/policySetDefinitions/Enforce-Mandatory-Tags",
+    "scope": "${current_scope_resource_id}",
+    "enforcementMode": null
+  },
+  "location": "${default_location}",
+  "identity": {
+    "type": "SystemAssigned"
+  }
+}
+```
+
+## Make the Custom Policy Definitions and Policy Set Definition available for use
+
+You now need to save your custom policy and policy set definitions at the `es_root` Management Group to ensure they can be used at that scope or any scope beneath. To do that, we need to extend the built-in archetype for `es_root`. 
```

Comment:
As per the linting error, need to remove trailing whitespace from this line please.

```shell
/github/workspace/docs/wiki/[Examples]-Create-Custom-Policy-and-Assignments.md:437:232 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

Reply:
Thanks Kevin, this should be resolved in the latest commit.